.github/ISSUE_TEMPLATE/release-checlist.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Release Checklist
+about: Template issue for creating release checklists.
+title: Checklist for vX.Y.Z release
+labels: ''
+assignees: ''
+
+---
+
+# Release Checklist
+
+This is the release checklist for Zeitgeist vX.Y.Z. All following checks should be completed before publishing a new release of the Zeitgeist runtime or client.
```

Comment:
Can be automated in a workflow this way by settings a VERSION environment variable

```suggestion
This is the release checklist for Zeitgeist {{ env.VERSION }}. All following checks should be completed before publishing a new release of the Zeitgeist runtime or client.
```